diff -Nru cmatrix-1.1a/Makefile.am cmatrix-1.1a.new/Makefile.am
--- cmatrix-1.1a/Makefile.am	Tue Apr  4 03:43:44 2000
+++ cmatrix-1.1a.new/Makefile.am	Tue Apr  4 09:31:08 2000
@@ -7,29 +7,8 @@
 		TODO install-sh missing mkinstalldirs matrix.fnt \
 		matrix.psf.gz mtx.pcf
 
-install-data-local:
-	@if test -d /usr/share/consolefonts; then \
-	    echo "Installing matrix fonts in /usr/share/consolefonts..."; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.fnt /usr/share/consolefonts; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz /usr/share/consolefonts; \
-	fi
-	@if test -d /usr/lib/kbd/consolefonts; then \
-	    echo "Installing matrix fonts in /usr/lib/kbd/consolefonts..."; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.fnt /usr/lib/kbd/consolefonts; \
-	    $(INSTALL_DATA) $(srcdir)/matrix.psf.gz /usr/lib/kbd/consolefonts; \
-	fi
-	@if test -d /usr/lib/X11/fonts/misc; then \
-	    echo "Installing X window matrix fonts in /usr/lib/X11/fonts/misc..."; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/lib/X11/fonts/misc; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/lib/X11/fonts/misc; \
-	    echo "Running mkfontdir /usr/lib/X11/fonts/misc..."; \
-	    $(MKFONTDIR) /usr/lib/X11/fonts/misc; \
-	else \
-	if test -d /usr/X11R6/lib/X11/fonts/misc; then \
-	    echo "Installing X window matrix fonts in /usr/X11R6/lib/X11/fonts/misc..."; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/X11R6/lib/X11/fonts/misc; \
-	    $(INSTALL_DATA) $(srcdir)/mtx.pcf /usr/X11R6/lib/X11/fonts/misc; \
-	    echo "Running mkfontdir /usr/X11R6/lib/X11/fonts/misc..."; \
-	    $(MKFONTDIR) /usr/X11R6/lib/X11/fonts/misc; \
-	fi \
-	fi
+consolefonts_DATA = matrix.fnt matrix.psf.gz
+consolefontsdir = /usr/share/consolefonts
+
+Xmiscfonts_DATA = mtx.pcf
+Xmiscfontsdir=/usr/X11R6/lib/X11/fonts/misc
